shift_detector/detector.py

```diff
@@ -25,6 +25,7 @@ def __init__(self,
                  df1: Union[pd.DataFrame, str],
                  df2: Union[pd.DataFrame, str],
                  delimiter=',',
+                 log_print=True,
```

Comment:
Is this our wanted behaviour? I would recommend to set the default to false. When the customer wants to get more information he can set the variable.

Otherwise I don't think that this option will ever be used...

Reply:
It is wanted behavior for the tests.

Reply:
The default should be true as it is used by most users and therefore should be as descriptive as possible. That's why it makes sense to turn the logs off during the tests as they don't convey any additional; value there.